components/export/ui/src/main/resources/PhenoTips/ExportFilter.xml

```diff
@@ -199,17 +199,39 @@
   #set ($query = ", StringProperty externalId $query and externalId.id.id = obj.id and externalId.id.name = 'external_id' and upper(externalId.value) like upper(?)")
   #set ($discard = $parameters.add("$request.get('external_id')%"))
 #end
-#set ($patientIds = $services.query.hql($query).bindValues($parameters))
-#if ($request.count)
-  #set ($discard = $patientIds.addFilter('count'))
-  $patientIds.execute()[0]
-  #break
-#end
-#set ($patientIds = $patientIds.execute())
-#if ($request.list)
-  $response.setContentType('application/json')
-  $jsontool.serialize($patientIds)
-  #break
+#set($visibilities = $request.getParameterValues('visibility'))
```

Comment:
pull changes from master into this branch. I made changes to this part of master.